components/style/context.rs

```diff
@@ -103,14 +103,28 @@ fn get_env_usize(name: &str) -> Option<usize> {
     })
 }
 
+/// A global variable holding the state of
+/// `StyleSystemOptions::default().disable_style_sharing_cache`.
+/// This is motivated by [issue #22854](https://github.com/servo/servo/issues/22854).
```

Comment:
I don't think this line of the comment is useful, it only makes sense in the context of this PR.

Reply:
I thought I'd try to leave the issue link for motivation especially because this is an odder pattern than usual, to try and preempt "why not just use `opts::get()`" for people not familiar with the issue.

Reply:
I still think it's not particularly useful. I assume we'd get rid of `opts` as a dependency of `style` after changing the other similar uses. Once that's done, if somebody wants to use `opts::get()` again they need to re-introduce it as a dependency, which makes the consequences of it much more obvious.

Not going to block this patch on removing this comment, but I don't quite see the point of it... Maybe just `See #22854` is a good compromise?

Either way this looks ok, mind squashing? Thanks!

Reply:
K, I'll do that and squash once I'm at a computer.